### PR TITLE
ResNet18 || CIFAR10 || bkj

### PR DESCRIPTION
**Pull request by bkj**

### CODE REVIEW

File: CIFAR10/train/basenet.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    "version": "v1.0",
+    "author": "bkj",
+    "authorEmail": "ben@canfield.io",
+    "framework": "pytorch",
+    "codeURL": "https://github.com/bkj/basenet/tree/49b2b61e5b9420815c64227c5a10233267c1fb14/examples",
+    "model": "Resnet18 + minor modifications",
+    "hardware": "V100 (AWS p3.2xlarge)",
+    "costPerHour": 3.060,
+    "timestamp": "2018-04-20",
+    "misc": {
```
Comment (codyaustun) [inline CIFAR10/train/basenet.json:11]: What hyperparameters did you use? Please add the following information to `misc`:

- learning rate schedule
- batch size
- optimizer
- momentum
- weight decay

The [Google ResNet50 submission](https://github.com/stanford-futuredata/dawn-bench-entries/blob/master/ImageNet/train/google_resnet50_tpu_tensorflow18.json) is a good example for reference.

File: CIFAR10/train/basenet.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "version": "v1.0",
+    "author": "bkj",
+    "authorEmail": "ben@canfield.io",
+    "framework": "pytorch",
+    "codeURL": "https://github.com/bkj/basenet/tree/49b2b61e5b9420815c64227c5a10233267c1fb14/examples",
```
Comment (codyaustun) [inline CIFAR10/train/basenet.json]: Please add the command line or instructions on how to reproduce your experiment to a README for this directory or add a `commandLine` field under `misc`

File: CIFAR10/train/basenet.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+    "version": "v1.0",
+    "author": "bkj",
+    "authorEmail": "ben@canfield.io",
+    "framework": "pytorch",
```
Comment (codyaustun) [inline CIFAR10/train/basenet.json]: What version of PyTorch did you use? Please update with the version number or git commit hash.

**Review by codyaustun — CHANGES_REQUESTED** [review]

File: CIFAR10/train/basenet.json (hunk 3 above)
Comment (bkj) [reply CIFAR10/train/basenet.json]: fixed

File: CIFAR10/train/basenet.json (hunk 1 above)
Comment (bkj) [reply CIFAR10/train/basenet.json:11]: added hyperparams

**Review by codyaustun — APPROVED** [review]
